docs/velox-backend-support-progress.md

```diff
@@ -88,11 +86,455 @@ Gluten supports 30+ operators (Drag to right to see all data types)
 | Velox2Arrow                 | Convert Velox format to Arrow format                                                                                                                                        | Y                                                      | Y                     | S       | S    | S     | S   | S    | S     | S      | S      | NS   | S      | S     | S   | S           | S    | NS        | S       | NS       | NS  |
 | WindowGroupLimitExec        | Optimize window with rank like function with filter on it                                                                                                                   | Y                                                      | Y                     | S       | S    | S     | S   | S    | S     | S      | S      | NS   | S      | S     | S   | S           | S    | NS        | S       | NS       | NS  |
 
-### Function support
+#### Scalar Functions Support Status
```

Comment:
should be 3 #

all belows headers are one level less

Reply:
Looks the header level all are wrong. Can you fix?

operator map should be second level

Scalar Functions Support Status is also 2nd level